ui/summary_test.go

```diff
@@ -164,3 +166,80 @@ func createTestMetrics() map[string]*stats.Metric {
 
 	return metrics
 }
+
+func TestSummarizeMetricsJSON(t *testing.T) {
+	metrics := createTestMetrics()
+	expected := `{
+    "root_group": {
+        "name": "",
+        "path": "",
+        "id": "d41d8cd98f00b204e9800998ecf8427e",
+        "groups": {
+            "child": {
+            "name": "child",
```

Comment:
Nit: could you fix the indentation here? (lines 179-189 are missing a `\t`)

Reply:
Not sure why `\t` here, I use 4 tabs for this JSON ident.

Reply:
Ah, these aren't `\t` chars, but my point still stands: lines 179-189 should be indented one more level.

Reply:
Fixed.

Reply:
Sorry, it still wasn't fixed, so I took the liberty in d63c5012. Just squash it before merge.